server_admin/topics/clients/oidc/con-secret-rotation.adoc

```diff
@@ -0,0 +1,28 @@
+[id="con-secret-rotation_{context}"]
+[[_secret_rotation]]
+= Client Secret Rotation
+[role="_abstract"]
+For client with <<_client-credentials, Confidential>> <<_access-type, Access Type>> {project_name} supports the functionality of rotating client secrets through <<_client_policies, Client Policies>>.
```

Comment:
Client secret rotation is preview feature and this fact probably worth to be mentioned in the beginning of this section. There is some snippet, which you can include to automatically address this. You can see for example "recovery-codes.adoc" for the inspiration when the similar snippet is used for the "Recovery codes" preview feature.

Reply:
thanks @mposolda